ui/test/hello_react_test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
Should this file be saved as `hello_react_test.jsx` instead of `hello_react_test.js`?